config/music.json

```diff
@@ -0,0 +1,43 @@
+[
+	{
+		"title":"Endless Space",
+		"artist":"Solus",
+		"license":"/datum/license/ccby3",
+		"audio_file":"sound/ambience/title0.mp3",
+		"url":"https://www.newgrounds.com/audio/listen/74946",
+		"jukebox":1,
+		"roundend":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Flip-Flap",
+		"artist":"X-CEED",
+		"audio_file":"sound/ambience/title1.mp3",
+		"url":"https://aminet.net/package/mods/xceed/Flipflap",
+		"license":"/datum/license/grandfathered",
+		"jukebox":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Robocop Gameboy Title Remix",
+		"artist":"CBoyardee",
+		"audio_file":"sound/ambience/title2.mp3",
+		"url":"https://archive.org/details/robocop_20160928",
+		"license":"/datum/license/grandfathered",
+		"jukebox":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Tintin on the Moon Remix",
+		"artist":"Cuboos (Original Artist: Jeroen Tel)",
+		"url":"https://www.youtube.com/watch?v=Hz6KO0gnrS8&t=0s",
+		"license":"/datum/license/grandfathered",
+		"audio_file":"sound/ambience/title3.mp3",
+		"jukebox":1,
+		"lobbymusic":1
+	},
+	{
+		"url":"https://www.youtube.com/watch?v=G2gVAPKlgqA",
+		"jukebox":1
+	}
```

Comment:
Why doesn't this one have all the parts filled in?

Reply:
It doesn't need the other parts because they are pulled in from the youtube DLL if loaded from a youtube path. This was more of an example and I should probably remove it to not encourage youtube URLs being default songs since they won't load from the server.

Reply:
I think having the title and license defined at a minimum would be best.